x-pack/filebeat/input/httpjson/input.go

```diff
@@ -37,6 +38,9 @@ const (
 
 var userAgent = useragent.UserAgent("Filebeat")
 
+// for testing
+var timeNow = time.Now
```

Comment:
Do we still need this one? And does it need also to be UTC?

Reply:
it is used to be able to swap it for a mock one during unit tests and return a fixed time of our choice, `UTC` is called on the `Time` object it returns, and it is done where this is called, does it make sense? I guess we could change the private var approach but for the given case I considered it a convenient one, wdyt?

Reply:
I think it should be okay yeah